### PR TITLE
fix: to inject app env script as one liner

### PR DESCRIPTION
**Pull request by emmenko**
No idea why this started "failing" now but in the [graphql-explorer Custom Application](https://github.com/commercetools/merchant-center-graphql-explorer) (built with Vite) the deployment was broken as the CSP policy refused to execute the injected script `window.app = ` due to a mismatch with the SHA hash.

After some debugging and try/error, it appeared that the browser expected the hash to be created from a one-liner of the script.

In other words, instead of

```
<script>
  window.app = 
</script>
```

it should be

```
<script>window.app = </script>
```

Update: it looks like this was introduced due to Prettier in #3379 

image

### CODE REVIEW

File: packages/mc-html-template/html-docs/application.html [hunk 1]
```diff
@@ -166,9 +166,7 @@
     __LOADING_SCREEN_JS__
 
     <!-- Application globals -->
-    <script>
-      window.app = __APPLICATION_ENVIRONMENT__;
-    </script>
+    __APPLICATION_ENVIRONMENT__
```
Comment (emmenko) [inline packages/mc-html-template/html-docs/application.html:169]: This now ensures that prettier doesn't mess with the format

**Review by CarlosCortizasCT — APPROVED** [review]
It must have been quite difficult to figure this out.

Awesome work! 💯

**Comment by emmenko** [comment]
Lots of debugging yes 😅

File: packages/mc-html-template/html-docs/application.html (hunk 1 above)
Comment (tdeekens) [reply packages/mc-html-template/html-docs/application.html:169]: Wow. That's quite a find.
